backend/src/main/java/clf/integra/backend/model/User.java

```diff
@@ -0,0 +1,24 @@
+package clf.integra.backend.model;
+import lombok.*;
```

Comment:
I would recommend not importing the whole library, but only what you're using. You can configure this in IntelliJ settings, add a high value